elevator.js

```diff
@@ -161,16 +171,7 @@ var Elevator = function(options) {
     }
 
     function bindElevateToElement( element ) {
-        if( element.addEventListener ) {
```

Comment:
For some reason that didn't work in my browser (google-chrome on Ubuntu), so I replaced it with pure jquery.
If you want this to be completely jquery free let me know, so I can adjust the PR. I tried to fix it for a bit, but I needed a quick and dirty solution :)